Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1797,13 +1797,7 @@ void webkit_settings_set_enable_offline_web_application_cache(WebKitSettings* se
 {
     g_return_if_fail(WEBKIT_IS_SETTINGS(settings));
 
-    WebKitSettingsPrivate* priv = settings->priv;
-    bool currentValue = priv->preferences->offlineWebApplicationCacheEnabled();
-    if (currentValue == enabled)
-        return;
-
-    priv->preferences->setOfflineWebApplicationCacheEnabled(enabled);
-    g_object_notify_by_pspec(G_OBJECT(settings), sObjProperties[PROP_ENABLE_OFFLINE_WEB_APPLICATION_CACHE]);
+    // NO OP
```

Comment:
We probably don't need this comment as no-op is obvious; `//Deprecated` may be more useful.
And I think these changes may need to be reviewed by glib reviewers, as I am not sure if they currently have clients that depend on these APIs, maybe @carlosgcampos @mcatanzaro?

Reply:
You did the right thing: keep the APIs, but make them no-ops. I doubt anything relies on this feature.

Please mark them as deprecated, though:

 * Add `Deprecated: 2.44` to the bottom of the doc comments on these functions.
 * Change `WEBKIT_API` to `WEBKIT_DEPRECATED` on the declarations in WebKitSettings.h.in
 * In  webkit_settings_class_init(), find `WebKitSettings:enable-offline-web-application-cache`, add `Deprecated: 2.44` and change the final argument to `static_cast<GParamFlags>(readWriteConstructParamFlags | G_PARAM_DEPRECATED)`